get-started/index.md

```diff
@@ -39,7 +39,7 @@ Elasticsearch supports diverse use cases. Select a solution and follow its dedic
 
 ## Next steps
 
-For learn more about our products and solutions, see:
+To learn more about our products and solutions, see:
```

Comment:
"see" is one of the terms that's discouraged in our guidelines. Typically use "check out", "view", or "refer" instead
```suggestion
To learn more about our products and solutions, check out:
```